lib/utils.js

```diff
@@ -75,8 +75,8 @@ var utils = module.exports = {
     // (the first being args and the second options) and with known
     // option keys in the first so that we can warn the user about it.
     if (optionKeysInArgs.length > 0 && optionKeysInArgs.length !== argKeys.length) {
-      console.warn( // eslint-disable-line no-console
-        'Stripe: Options found in arguments (' + optionKeysInArgs.join(', ') + '). Did you mean to pass an options ' +
+      emitStripeWarning( // eslint-disable-line
```

Comment:
Maybe just call it `emitWarning` — I don't think we need the "Stripe" in there.

Also, now that there's no more console call here, can we remove the `eslint-disable-line`?

Reply:
Yes, and yes.  :)